docs/_docs/playout-history.md

```diff
@@ -32,14 +32,13 @@ The **History Templates** page on the History menu enables you to prepare report
 
 ![](/img/new-hist-temp.png)
 
-Either of these actions opens a page in which you can name the new template, and add or remove elements from the list on the left. To add a new element from the list on the right, click the plus icon for the item you require. If the element you require is not listed, you can use the **Add New Field** box at the lower end of the right side column. Select *string*, *boolean*, *integer*, or *float*, depending on the type of data that you wish to log, and then click the **+ Add** button.
+Either of these actions opens a page in which you can name the new template, and add or remove elements from the list on the left. To add a new element from the list on the right, click the plus icon for the item you require. If the element you require is not listed, you can use the **Add New Field** box at the lower end of the right side column. Select _string_, _boolean_, *integer*, or _float_, depending on the type of data that you wish to log, and then click the **+ Add** button.
```

Comment:
Ok, this is a nice one, I found a bunch of weird space chars here and there:

![image](https://user-images.githubusercontent.com/19195485/120172364-40a24980-c203-11eb-978e-4a0609d1539c.png)

After replacement, everything works as expected (prettier didn't change it because of that char).